docs/code-examples/tensor_simple.py

```diff
@@ -1,12 +1,10 @@
 """Create and log a tensor."""
-
+import numpy as np
 import rerun as rr
-from numpy.random import default_rng
 
-rng = default_rng(12345)
```

Comment:
I figured why set a seed in python if we don't for any of the other languages and they don't produce the same value any way